README.md

```diff
@@ -114,6 +114,10 @@ Here are some resources to get you started with Earthly
   * [C++](./examples/cpp)
   * [Mono-repo](./examples/monorepo)
   * [Multi-repo](./examples/multirepo)
+  * [Python](./examples/python)
+  * [Ruby](./examples/ruby)
+  * [Rust](./examples/rust)
+  * [Terraform](./examples/terraform)
```

Comment:
The terraform example isn't very reliable. I think it's better not to feature it.

Also, the examples should be ordered better - e.g. the language examples should sit together.

Reply:
I have sorted them alphabetically as per @alexcb comment [here](https://github.com/earthly/earthly/pull/1253/files#r721852823). I have also removed the terraform example like you mentioned.